packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -483,30 +478,124 @@ export class PerformanceEvent {
     }
 }
 
+// Note - these Telemetry types should move to common-definitions package
+export enum TelemetryDataTag {
+    None,
```

Comment:
Unsure, but I wanted an explicit case for the untagged approach - but you're right, it's redundant as-written since you can just use a literal instead of wrapping. I like that the 0 value for the enum means none.